SubredditService/Sources/SubredditDataSource.swift

```diff
@@ -4,12 +4,14 @@ import ListingService
 
 public class SubredditDataSource {
     public private(set) var subreddits = [Subreddit]()
+    private var pagingFinished: Bool
     public var updated: (() -> Void)?
 
     private var refreshTask: URLSessionTask?
     private var loadMoreTask: URLSessionTask?
 
     public init() {
+        self.pagingFinished = false
```

Comment:
Can you change the definition above `private var pagingFinished = false` instead?